rdkcentral/Dobby
# RDK-32028: Add "Thunder" RDK Plugin

### Description
Adds the `Thunder` RDK Plugin to automatically configure a container to access Thunder and generate security tokens if required.

### Test Procedure
Create a container with the Thunder plugin in the `rdkPlugins` section - see the README.md for details.

Container should start with no errors and should be able to access Thunder services on the host. If securityagent is running and a `bearerUrl` is set, then create a token and make available to the container with the `THUNDER_SECURITY_TOKEN` environment variable.

### Type of Change

### Requires Bitbake Recipe changes?
